### PR TITLE
TVL

### PR DESCRIPTION
**Pull request by lazydefi1**

### CODE REVIEW

**Comment by lazydefi1** [comment]
Screenshot 2021-09-15 at 12 43 26

File: src/views/FarmInfo.jsx [hunk 1]
```diff
@@ -34,12 +35,13 @@ export default function FarmInfo({
   //props{match.params, provider, userSigner, address, tx}
   function showFarm() {
     const params = useParams();
-    console.log("showFarm!");
-    console.log("params: ", params);
+    //console.log("showFarm!");
```
Comment (luffyowls) [inline src/views/FarmInfo.jsx:38]: Remove

**Review by luffyowls — COMMENTED** [review]
lgtm

File: src/App.jsx [hunk 2]
```diff
@@ -244,6 +257,16 @@ function App(props) {
...
 
+  /* compute NYAN/ETH price */
+  /* 1) query NYAN token contract for LP balance */
+  const nyanTokenInstance = useExternalContractLoader(injectedProvider, NyanTokenAddress, NyanTokenAbi);
+  const nyanLPBalance = useContractReader({ NyanToken: nyanTokenInstance }, "NyanToken", "balanceOf", [NyanETHSushiLPAddress]);
+  /* 2) query WETH token contract for LP balance */
+  const wethTokenInstance = useExternalContractLoader(injectedProvider, WETHTokenAddress, ERC20Abi);
+  const wethLPBalance = useContractReader({ ERC20: wethTokenInstance }, "ERC20", "balanceOf", [NyanETHSushiLPAddress])
+  /* we mul by 2 because an AMM pool is 50% tokenA + 50% tokenB */
+  const nyanETHPrice = (wethLPBalance && nyanLPBalance) ? parseFloat(2 * formatEther(wethLPBalance)) / parseFloat(formatEther(nyanLPBalance)) : undefined;
```
[the remark this answers is not in the document]
Comment (luffyowls) [reply src/App.jsx:268]: TIL JS handles this implicitly, all good

**Pull request merged**